src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.ts

```diff
@@ -51,6 +54,29 @@ export class InstructorHelpQuestionsSectionComponent extends InstructorHelpSecti
     showRecipientNameTo: [FeedbackVisibilityType.INSTRUCTORS, FeedbackVisibilityType.RECIPIENT],
   };
 
+  readonly exampleNumericalScaleEditFormModel: QuestionEditFormModel = {
```

Comment:
Just a nit. The order of fields can be more logically (even it does not matter in JavaScript).

- Like the `customNumberOfEntitiesToGiveFeedbackTo` should be together with `numberOfEntitiesToGiveFeedbackToSetting`. 
- Form control fields (`isEditable`, `isSaving`) can be separated from the actual form fields.

Reply:
Reordered fields for all question model for readability.